### ISSUE TITLE
no-spaced-func tends to report on the wrong line with long function chains

### ISSUE THREAD

**Issue by bgw** [eslint/eslint#3289]
Given code like:

```js
return _(foo)
  .map((el) => ...)
  .uniq()
  .value ();
```

Here the error is on the last line, but since `no-spaced-func` reports on the function callee, it reports on the `MemberExpression` and the output shows the first line.

Instead if the callee is a `MemberExpression`, it should report on `node.callee.property`, or alternatively `lastCalleeToken`.

**Edit**: Yes, I'd be willing to work on this.

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by mysticatea** [comment]
Looks good to me.

**Comment by nzakas** [comment]
:+1:
